pom.xml

```diff
@@ -0,0 +1,308 @@
+                        <Import-Package>
+                            org.apache.commons.lang;version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging;version="${commons-logging.osgi.version.range}",
+
+                            org.apache.http;version="${httpcomponents-httpcore.osgi.version.range}",
+                            org.apache.http.auth;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.client;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.client.config;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.client.methods;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.client.protocol;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.config;version="${httpcomponents-httpcore.osgi.version.range}",
+                            org.apache.http.conn;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.conn.routing;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.conn.socket;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.conn.ssl;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.entity;version="${httpcomponents-httpcore.osgi.version.range}",
+                            org.apache.http.impl.auth;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.impl.client;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.impl.conn;version="${httpcomponents-httpclient.osgi.version.range}",
+                            org.apache.http.params;version="${httpcomponents-httpcore.osgi.version.range}",
+                            org.apache.http.pool;version="${httpcomponents-httpcore.osgi.version.range}",
+                            org.apache.http.protocol;version="${httpcomponents-httpcore.osgi.version.range}",
+
+                            org.wso2.carbon.utils;version="${carbon.kernel.version.range}",
+                            org.wso2.securevault;version="${org.wso2.securevault.version.range}",
+                            org.wso2.securevault.definition;version="${org.wso2.securevault.version.range}",
+                            org.wso2.securevault.keystore;version="${org.wso2.securevault.version.range}",
+                            org.wso2.securevault.secret;version="${org.wso2.securevault.version.range}",
+                        </Import-Package>
+                        <!--                    Only the dependencies that are required by the AWS SDK which are not available in the IS are embedded.-->
```

Comment:
Shall we remove unwanted empty spaces. Please check other places as well

Reply:
Done